gladly_apis/actions/actions_schema.graphql

```diff
@@ -56,6 +96,9 @@ input LinkInput {
 type Query {
     inboxes(enabledOnly: Boolean): [Inbox] @action(name: "inboxes")
     topics(enabledOnly: Boolean): [Topic] @action(name: "topics")
+    lookupCustomerById(customerId: String!): Customer @action(name: "lookup_customer_by_id")
+    lookupCustomersByEmail(email: String!): [Customer] @action(name: "lookup_customers_by_email")
```

Comment:
You should combine `lookup_customers_by_email` and `lookup_customers_by_phone` in to one action named something like `lookup_customers_by_email_phone` since the response is identical for the two.

When generating the URL you can check for the presence of `.inputs.email` or `.inputs.phoneNumber` to determine which query value to use.